docs/danews.md

```diff
@@ -0,0 +1,147 @@
+
+*license*: Not publicly available.
+
+---
+
+DaNews consist of articles collected from the [Infomedia](https://infomedia.dk) and 
+consists of 25874862 articles across 1362 news sources. DaNews include
+articles from the period 01/12/2000 to  30/04/2021.
+
+DaNews have been filters out quality using a series of heuristic filters and removing repitious texts. 
+Following the filtering DaNews is deduplicated to remove exact and near-duplicates. For more on data cleaning
+see post processing.
+
+DaNews includes a total of 9296359450 tokens of which 8667100588 (0.93%) are left after filtering.
+
+## Datasheet:
+
+### Motivation:
+**For what purpose was the dataset created? Who created the dataset? Who funded the creation of the dataset?**
+
+The dataset was created for pre-training Danish language models by a team of researchers at Center for Humanities
+Computing Aarhus (CHCAA) using a codebase jointly developed with partners from industry (e.g. KMD, Ekstra Bladet) and other
+research institutions (e.g. Briston University, Alexandra Institute). For more on collaborators on this project see
+the [GitHub repository](https://github.com/centre-for-humanities-computing/danish-foundation-models
+).
+
+**Any other comments?**
+No.
+
+## Composition
+**What do the instances that comprise the dataset represent (e.g., documents, photos, people, countries)?**
+
+All instances of the dataset are Danish articles depending on the source these are derived from tabloid media or 
+mainstream news. 
+
+**How many instances are there in total (of each type, if appropriate)?**
+The dataset consist of 25874862 documents where 1048815 (0.04%) are left after filtering. 
+
+**Does the dataset contain all possible instances or is it a sample (not necessarily random) of instances from a larger set?**
+Prior to filtering this dataset contains all instances of new articles from the given period across the sources.
+
+**If the dataset is a sample from a larger set, what was the sampling strategy?**
+The dataset is not a sample, however it is filtered as described in post processing.
+
+**Who was involved in the data collection process?**
+A team of researchers at Center for Humanities
+Computing Aarhus (CHCAA) recieved this dataset in a collaboration with Infomedia.
+
+**Over what timeframe was the data collected?**
+The dataset include articles from the period 01/12/2000 to  30/04/2021.
+
+**Were any ethical review processes conducted?**
+No.
+
+## Preprocessing/cleaning/labeling
+
+**Was any preprocessing/Cleaning/Labeling of the data done 
+(e.g., discretization or bucketing, tokenization, part-of-speech tagging, 
+SIFT feature extraction, removal of instances, processing of missing values)?**
+
+We filter documents that are not in Danish. We remove low-quality text and deduplicate documents based on their 
+document-level n-gram similarity. The full pre-processing details are given in the post processing section.
```

Comment:
I specified the language detection algorithm. I'm not sure if that's the one we're using, so feel free to correct if necessary.
```suggestion
We removed non-Danish documents, according to the FastText language detection algorithm. We also remove low-quality text and deduplicate documents based on their 
document-level n-gram similarity. The full pre-processing details are given in the post processing section.
```

Reply:
We actually don't filter these documents for language (will fix this)